File: Inverted triangle of stars/Inverted_triangle_of_stars.java
class Solution {
    static String[] invIsoTriangle(int n) {
        // in this question in one row there are 2*n-1 element
        // and each row have spaces in starting and ending
        // in zero based indexing no. of space are equal to row no.
        String ans[] = new String[n];
        for(int i =0; i<n; i++) {
            String temp = "";
            // space
            for(int j =0; j<i; j++) {
                temp += " ";
            }
            // star
            for(int j = 2*(n-i)-1; j>0; j--) {
                temp += "*";
            }
            // space
            for(int j =0; j<i; j++) {
                temp += " ";
            }
            ans[i] = temp;
        }
        return ans;
    }
};

File: Inverted triangle of stars/Inverted_triangle_of_stars.py
class Solution:
    def invIsoTriangle(self, n):
        # in this question in one row there are 2*n-1 element
        # and each row have spaces in starting and ending
        # in zero based indexing no. of space are equal to row no.
        ans = []
        for i in range(0,n):
            temp = ""
            # space
            for j in range(0,i):
                temp += " "
           
            # star
            for j in range(2*(n-i)-1,0,-1):
                temp += "*"
        
            # space
            for j in range(0,i):
                temp += " "
            
            ans.append(temp)
        
        return ans; 
